pkgs/development/python-modules/pygls/default.nix

```diff
@@ -36,7 +36,16 @@ buildPythonPackage rec {
   ];
 
   propagatedBuildInputs = [
-    lsprotocol
+    (lsprotocol.overridePythonAttrs (old: {
```

Comment:
You will also have to add a thing for the update script to not try to update this version of the package